src/main/java/org/folio/des/scheduling/quartz/converter/bursar/ExportConfigToBursarJobDetailConverter.java

```diff
@@ -0,0 +1,34 @@
+package org.folio.des.scheduling.quartz.converter.bursar;
+
+import static org.folio.des.scheduling.quartz.QuartzConstants.EXPORT_CONFIG_ID_PARAM;
+import static org.folio.des.scheduling.quartz.QuartzConstants.TENANT_ID_PARAM;
+
+import org.folio.des.domain.dto.ExportConfig;
+import org.folio.des.scheduling.quartz.converter.ExportConfigToJobDetailConverter;
+import org.folio.des.scheduling.quartz.job.bursar.BursarJob;
+import org.quartz.JobBuilder;
+import org.quartz.JobDetail;
+import org.quartz.JobKey;
+import org.springframework.stereotype.Component;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+
+@Component
+@Log4j2
+@RequiredArgsConstructor
+public class ExportConfigToBursarJobDetailConverter implements ExportConfigToJobDetailConverter {
```

Comment:
this class looks very similar to ExportConfigToEdifactJobDetailConverter, only job class differ. Could one class with job class passed be used instead?